src/Octoshift/Handlers/CreateTeamCommandBaseHandler.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using OctoshiftCLI.Commands;
+using OctoshiftCLI.Contracts;
+
+namespace OctoshiftCLI.Handlers;
+
+public class CreateTeamCommandBaseHandler
```

Comment:
nit: I would rename this to just `CreateTeamCommandHandler`, because having `Base` kinda indicates that this is supposed to be subclassed but it's not the case here. The fact that these are in our shared project means that these are shared handlers already.

Reply:
Yeah I went back and forth on this. I'm gonna change it.